.github/workflows/ci.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Setup Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
-          node-version: 12
+          node-version: '14.x'
```

Comment:
We should use Node 16 as much as possible now.

Reply:
I agree.